go.mod

```diff
@@ -4,10 +4,11 @@ go 1.14
 
 require (
 	github.com/davecgh/go-spew v1.1.1
+	github.com/go-git/go-git/v5 v5.1.0
```

Comment:
Why not use go-getter to get the git repo?

Reply:
I thought about it. go-getter uses CLI execution of `git`, and a lot of code around SSH key handling and other things which I felt was overkill. I guess this strategy is already requiring Go on your PATH, but trying to keep the requirements minimal.